versioned_docs/version-8.6/components/connectors/protocol/rest.md

```diff
@@ -203,6 +203,12 @@ Secrets are currently not supported in the body of a **REST Connector**.
 }
 ```
 
+### Encoding
+
+In certain scenarios, such as when working with APIs that require pre-encoded URL elements, the REST Connector's default behavior may inadvertently modify encoded segments.
+
+To avoid this, set the `skipEncoding` value to `"true"` in the element template XML. This disables the automatic decoding and re-encoding process, ensuring the URL is sent to the server exactly as provided. This flag is available since 8.6.10.
+
```

Comment:
Should this change also be made in the 8.7 and 8.8 docs?

Reply:
Thanks for having a look at this, as mentioned in the description of the PR, this PR is only for documenting the backport to 8.6. as the original documentation was done [here](https://github.com/camunda/camunda-docs/pull/4837/) and i think this was merged before 8.8 was created, so this should be present in 8.7. and 8.8. already